src/commands/Hello.ts

```
@@ -0,0 +1,26 @@
+import {
+  CommandInteraction,
+  Client,
+  ApplicationCommandType,
+  InteractionResponse,
+} from 'discord.js';
+
+import { Command } from '../interfaces/Command';
+
+export const Hello: Command = {
+  name: 'hello',
+  description: 'Returns a greeting',
+  type: ApplicationCommandType.ChatInput,
+  run: (
+    client: Client,
+    interaction: CommandInteraction,
+  ): Promise<InteractionResponse<boolean>> => {
+    const username = interaction.user?.username;
```

can user ever be undefined here? we may want to throw an error in that case